apps/server/src/services/TimerService.ts

```diff
@@ -240,77 +245,95 @@ export class TimerService {
     }
 
     // force an update
-    this.update();
+    this.update(true);
   }
 
-  update() {
-    this.timer.clock = clock.timeNow();
+  private updateRoll() {
+    const tempCurrentTimer = {
```

Comment:
Since you have a type defined for `CurrentTimers`, maybe it's a good idea to type this object as such to prevent any ambiguities.